nucliadb/nucliadb/ingest/orm/knowledgebox.py

```diff
@@ -486,22 +486,23 @@ async def purge(cls, driver: Driver, kbid: str):
         await cls.delete_all_kb_keys(driver, kbid)
 
     @classmethod
-    async def delete_all_kb_keys(cls, driver: Driver, kbid: str):
-        # Delete KB Keys
+    async def delete_all_kb_keys(
+        cls, driver: Driver, kbid: str, chunk_size: int = 1_000
```

Comment:
@jotare I have added back the `chunk_size` parameter because it makes testing way easier.
Otherwise I have to create more than 1000 resources in a test and the test takes too long.